### ISSUE TITLE
runnumber is saved locally in RunNumber file

### ISSUE THREAD

**Issue by fbdescamps** [snoplus/orca#82]
This means, when changing control from operug to operag, the operator needs to go to the runcontrol window and set the runnumber through the runnumber panel manually. Can't this be saved in the .orca?

**Comment by tlatorre-uchicago** [comment]
Should be able to get this from the postgres database once that is merged into master.

**Comment by fbdescamps** [comment]
@tlatorre-uchicago @boardhead : quick reminder to  retire this issue.. I believe it was resolved with PR #162 ? ⛵️

**Comment by tlatorre-uchicago** [comment]
Solved by #162
